Fix: proza-message en map-select-action

ksvlaanderen opened the pull request:
fix: UIG-2454 - vl-proza-message - absolute path
fix: UIG-2486 - vl-map-select-action - typing issue

Goldflow reviewed (changes requested):
didn't run code but changes look good

kzou wel nog de storybook aanpassen 
> "Het Proza bericht zal opgehaald worden vanaf ***{baseUrl}proza***/domein/{domain}/{code}."

Done!

ksvlaanderen committed:
fix: UIG-2454 - vl-proza-message - absolute path

Slash verwijderd zodat door "/" mee te geven als baseUrl het absolute path gebruikt wordt

ksvlaanderen committed:
fix: UIG-2486 - vl-map-select-action - typing issue

 "Unknown" veranderd naar "any" om typing issue te fixen bij afnemers

Goldflow reviewed (approved):
Noice!

kspeltix commented:
:tada: This PR is included in version 1.10.0 :tada:

The release is available on [GitHub release](https://github.com/milieuinfo/uigov-web-components/releases/tag/v1.10.0)

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:
